bin/check-postgres-replication.rb

```diff
@@ -117,7 +117,13 @@ def run
                              sslmode: ssl_mode,
                              connect_timeout: config[:timeout])
 
-    master = conn_master.exec('SELECT pg_current_xlog_location()').getvalue(0, 0)
+    pg_vsn = conn_master.exec("SELECT current_setting('server_version')").getvalue(0, 0)
+
+    master = if Gem::Version.new(pg_vsn) > Gem::Version.new('9.6')
```

Comment:
I think it is safer to check if less than 10 and greater or equal to 9 as they may need to hotfix making a `9.6.1`.

Example:
```ruby
master = if Gem::Version.new(pg_vsn) < Gem::Version.new('10.0') && Gem::Version.new(pg_vsn) >= Gem::Version.new('9.0')
```

Reply:
:+1: